plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergInputInfo.java

```diff
@@ -22,16 +23,35 @@
 public class IcebergInputInfo
 {
     private final Optional<Long> snapshotId;
+    private final boolean partitioned;
+    private final String fileFormat;
 
+    @JsonCreator
     public IcebergInputInfo(
-            @JsonProperty("snapshotId") Optional<Long> snapshotId)
+            @JsonProperty("snapshotId") Optional<Long> snapshotId,
+            @JsonProperty("partitioned") boolean partitioned,
+            @JsonProperty("fileFormat") String fileFormat)
```

Comment:
In Iceberg, can different files be of different format?

Reply:
Yes, it is possible that's why updated it to `tableDefaultFileFormat`.